filament/src/details/View.h

```diff
@@ -17,6 +17,12 @@
 #ifndef TNT_FILAMENT_DETAILS_VIEW_H
 #define TNT_FILAMENT_DETAILS_VIEW_H
 
+// Avoid warnings for using the ToneMapping API, which has been publically deprecated.
+#if defined(__clang__)
```

Comment:
This will not fix visual studio right?

Reply:
Well, this warning silences warnings due to using APIs marked `__attribute(deprecated)`, which I don't think MSVC has.